<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorPanasonicPresetWidget</class>
 <widget class="QWidget" name="InspectorPanasonicPresetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAddress">
     <item>
      <spacer name="horizontalSpacerAddress">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAddress">
       <property name="text">
        <string>Address</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAddress">
       <property name="placeholderText">
        <string>IP address / Hostname</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPreset">
     <item>
      <spacer name="horizontalSpacerPreset">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>55</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPreset">
       <property name="text">
        <string>Preset</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPreset"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerOscOutput">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorPanasonicPresetWidget</receiver>
   <slot>addressChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPreset</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorPanasonicPresetWidget</receiver>
   <slot>presetChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorPanasonicPresetWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressChanged(QString)</slot>
  <slot>presetChanged(QString)</slot>
  <slot>triggerOnNextChanged(int)</slot>
 </slots>
</ui>
